src/libraries/System.IO.Pipes.AccessControl/pkg/System.IO.Pipes.AccessControl.pkgproj

```diff
@@ -0,0 +1,10 @@
+<Project DefaultTargets="Build">
+  <ItemGroup>
+    <ProjectReference Include="..\src\System.IO.Pipes.AccessControl.csproj" />
```

Comment:
You need to add a reference to the ref project and add supported frameworks:

https://github.com/dotnet/runtime/blob/master/src/libraries/System.CodeDom/pkg/System.CodeDom.pkgproj#L4-L6

Reply:
Actually it seems like we can leave this as is and it will use lib assemblies to compile as well.

Reply:
Yes but you were right that he has to add Supported frameworks in here. If not the build will fail.

Reply:
Yep.

Reply:
done